### ISSUE TITLE
DBeaver FR: Add display of SQLite BOOLEAN columns as checkboxes in data grid

### ISSUE THREAD

**Issue by pdanie** [dbeaver/dbeaver#7848]
DBeaver - **Version 6.3.4 CE**
DBeaver driver: **MS SQL Server  / Microsoft Driver** _(the new Microsoft Driver)_
Operating System: **Windows 7 / Windows 8.1 / Windows 10**
Database Server: **Microsoft SQL Express 2014, 2016, 2017**

It would be great if the system setting below made SQLite **BOOLEAN** column values presented as checkboxes instead of just **1** or **0** (and **NULL**).

Currently the system setting has no effect on the display in the result grids.
 
**Run the SQL script below**
```
CREATE TABLE TestBoolean (Id INTEGER, Flag BOOLEAN, Message TEXT);

INSERT INTO TestBoolean VALUES (1, TRUE, 'My Message');

SELECT * FROM TestBoolean;
```

**It result into (either way the system setting value)**
```
Id   Flag  Message
  1     1  My Message
```

**SystemSetting**:

The system setting in questing is:

&nbsp;&nbsp;**DBeaver** -> **Editors** -> **Data Editor** -> **Presentation** -> **Grid** -> **Show booleans as checkboxes**.
<br/>
**PS**. This is the same as CASE #7806 that wanted the feature for SQL Server databases.

**Comment by serge-rider** [comment]
SQLite doesn't support boolean data types (as well as many other databases).

But we could try to use column transformers (numeric->boolean).

**Comment by pdanie** [comment]
I know there is no BOOLEAN type in SQLite, only integers.

But if a column is declared as BOOLEAN (see that *DBeaver* present the table property BOOLEAN), a test like the one below ought to do the trick for that kind of column:

Value **= 0**  => Boolean **FALSE**
Value **<> 0**  => Bolean **TRUE**

**Comment by kseniiaguzeeva** [comment]
Seems, the feature affects on View/Format>Column Formats. This item of context menu cannot be opened.
![flOFE2u5w3](https://user-images.githubusercontent.com/51405061/75093592-2efbdc00-5594-11ea-95f6-1e4e009268cf.gif)
[dbeaver-debug.log](https://github.com/dbeaver/dbeaver/files/4239423/dbeaver-debug.log)

**Comment by serge-rider** [comment]
Implemented in #7927
